src/user/profile/popups/followers.js

```diff
@@ -8,74 +12,168 @@ export class Followers extends Component{
-                  <p className="descr">Front-End Developer</p>
-                </div></Col>
-                <Col md={3}><Button className="follow-link" variant="outline-primary">Follow</Button></Col>
-            </Row>
-            <Row className="follower" id="p1">
-                <Col md={2}><img class="photo" src={logo} alt="I"></img></Col>
-                <Col md={7}><div className="contain"> 
-                  <p className="user">Dhanus Rajendra</p>
-                  <p className="descr">Front-End Developer</p>
-                </div></Col>
-                <Col md={3}><Button className="follow-link" variant="outline-primary">Follow</Button></Col>
-            </Row>
-            <Row className="follower" id="p1">
-                <Col md={2}><img class="photo" src={logo} alt="I"></img></Col>
-                <Col md={7}><div className="contain"> 
-                  <p className="user">Dhanus Rajendra</p>
-                  <p className="descr">Front-End Developer</p>
-                </div></Col>
-                <Col md={3}><Button className="follow-link" variant="outline-primary">Follow</Button></Col>
-            </Row>
-            <Row className="follower" id="p1">
-                <Col md={2}><img class="photo" src={logo} alt="I"></img></Col>
-                <Col md={7}><div className="contain"> 
-                  <p className="user">Dhanus Rajendra</p>
-                  <p className="descr">Front-End Developer</p>
-                </div></Col>
-                <Col md={3}><Button className="follow-link" variant="outline-primary">Follow</Button></Col>
-            </Row>
-            <Row className="follower" id="p1">
-                <Col md={2}><img class="photo" src={logo} alt="I"></img></Col>
-                <Col md={7}><div className="contain"> 
-                  <p className="user">Dhanus Rajendra</p>
-                  <p className="descr">Front-End Developer</p>
-                </div></Col>
-                <Col md={3}><Button className="follow-link" variant="outline-primary">Follow</Button></Col>
-            </Row>
-            <Row className="follower" id="p1">
-                <Col md={2}><img class="photo" src={logo} alt="I"></img></Col>
-                <Col md={7}><div className="contain"> 
-                  <p className="user">Dhanus Rajendra</p>
-                  <p className="descr">Front-End Developer</p>
-                </div></Col>
-                <Col md={3}><Button className="follow-link" variant="outline-primary">Follow</Button></Col>
-            </Row>
-        </Modal.Body>
+        <Dialog
+          {...this.props}
+          aria-labelledby="customized-dialog-title">
+          <DialogTitle> 
+            <span className="title">Followers</span>
+            {this.props.onClose ? <IconClose/> : null}
+            <div className="about">PEOPLE WHO FOLLOW YOU</div>
+          </DialogTitle> 
+          <DialogContent>
+            <Grid container spacing={4}>        
```

Comment:
@theprover97 Just use a for loop, store name and designation object in an array.
We can reduce the repetition of the code.
Hope you understand what I am trying to say.